.github/workflows/ca-kryoptic-test.yml

```
@@ -0,0 +1,364 @@
+          diff expected actual
+
+          docker exec pki runuser -u pkiuser -- \
+              pki \
+              -d /var/lib/pki/pki-tomcat/conf/alias \
+              -f /var/lib/pki/pki-tomcat/conf/password.conf \
+              nss-cert-verify \
+              --cert-usage SSLServer \
+              HSM:sslserver
+
+      - name: Run PKI healthcheck
+        if: false
```

actionlint (1.7.12)

[error] constant expression "false" in condition. remove the if: section

(if-cond)